app/controllers/tas_controller.rb

```diff
@@ -17,6 +17,17 @@ def edit
     @user = Ta.find_by_id(params[:id])
   end
 
+  def destroy
+    @user = Ta.find_by_id(params[:id])
+    if @user.destroy
```

Comment:
I think this is a runtime error if the TA isn't found - you can flash an error in that case.